core/controllers/feedback_test.py

```diff
@@ -152,7 +158,7 @@ def test_create_thread(self):
         thread_url = '%s/%s/%s' % (
             feconf.FEEDBACK_THREAD_URL_PREFIX,
             self.EXP_ID,
-            threadlist[0]['thread_id'])
+            threadlist[0]['thread_id'].split(".")[1])
```

Comment:
ditto re single quotes, here and elsewhere.

Also, can we instead have a method in feedback_services that takes a combined id and splits it?

Finally, I'm a bit worried that thread id still seems to be referring to two different things. Can it refer to just the second segment, throughout the codebase, and maybe we use full_thread_id for the combined thing?